includes/definitions/discovery/stellar.yaml

```diff
@@ -0,0 +1,38 @@
+modules:
+    os:
+        hardware:
```

Comment:
You could also extract `version`, seems it's part of sysdescr. Or might be it's own Oid.

If it's only in sysdescr you can do like;

sysDescr_regex: '/\s(?<version>[\d\.]+)/'

Reply:
There is currently no way to get the full version via SNMP. In this case it would be 4.0.3.16 (4.0.3 being the Major release; 16 being the build number; which you can't see in the output). I'm in contact with our R&D to get a MIB/OID for this in the future, but would prefer to not "read out" the improper value now.

That aside, if you'd like to have the "incomplete" version number (that may be misleading to customers) - it could be added.